content/chainguard/libraries/python/build-configuration.md

```diff
@@ -99,6 +100,14 @@ for accessing your organization's Sonatype Nexus repository group.
    authentication is required, you must use the relevant details such as
    username and password in your build tool configuration.
 
+### Direct access
+
+Build configuration to retrieve artifacts directly from the Chainguard Libraries
+for Python repository at `https://libraries.cgr.dev/python/` with the simple
```

Comment:
```suggestion
for Python repository at `https://libraries.cgr.dev/python/`, with the simple
```

Reply:
First part is not a sentence if I break it up